examples/autodiff.jl

```diff
@@ -114,7 +114,7 @@ dbx[2] == 1.0
 # second-order model by computing the derivative of the gradient at multiple points at once.
 # We begin by defining a helper function for the gradient. Since we will not need the original results
 # (stored in y), we can mark it DuplicatedNoNeed. Specifically, this will perform the following:
-#      dx += dy * gradient f wrt x
+#      dx += dy * gradient f wrt x  
```

Comment:
I'm not sure this actually adds a line break. Regardless, would you consider updating this to LaTeX, like above?